cryk-ui/src/pages/CryptoInformation.tsx

```diff
@@ -150,26 +151,47 @@ export default function CryptoInformation(props: any) {
         )
     }
 
+    const symbol = cryptocurrency?.symbol;
+    const description = cryptocurrency?.description;
+    const website = cryptocurrency?.website;
+    const source = cryptocurrency?.source;
+    const reward = Number(cryptocurrency?.blockReward) < 0 ? null: Number(cryptocurrency?.blockReward);
+    const coins = Number(cryptocurrency?.totalCoins) < 0 ? null: Number(cryptocurrency?.totalCoins);
+
     return (
         <HelmetProvider>
             <div>
                 <Helmet>
                     <title>{title}</title>
                 </Helmet>
-                <div className="page-container">
-                    <div className="title">
-                        <h1>{cryptocurrency?.symbol}</h1>
+                <div className="page-container" vocab="http://purl.org/net/bel-epa/doacc#" typeof="Cryptocurrency">
+                    <div className="title" property="http://purl.org/net/bel-epa/doacc#title" 
+                    about="http://purl.org/net/bel-epa/doacc#{id}">
+                        <h1 property="http://purl.org/net/bel-epa/doacc#symbol">{symbol}</h1>
                     </div>
-                    <p className="crypto-description">
-                        {cryptocurrency?.description}
+                    <p className="crypto-description" property="http://purl.org/net/bel-epa/doacc#description"
+                        typeof="http://purl.org/net/bel-epa/doacc#description">
+                        {description}
+                    </p>
+                    <p className="crypto-website" property="http://purl.org/net/bel-epa/doacc#website"
+                        typeof="http://purl.org/net/bel-epa/doacc#WebSite">
+                        <span property="http://purl.org/net/bel-epa/doacc#name">Official website: </span>
```

Comment:
The doacc vocab does not contain http://purl.org/net/bel-epa/doacc#name, http://purl.org/net/bel-epa/doacc#url or http://purl.org/net/bel-epa/doacc#value